File: Data Structure & Algorithms/Patterns/Pattern 11/Pattern11_Solution.js

function printPattern(num) {
    let n = 1;
    for (let i = 1; i <= num; ++i) {
        for (let j = 1; j <= i; ++j) {
            process.stdout.write((n) + "\t");
            n++;
        }
        console.log();
    }
}

printPattern(5)

File: DSA_Level-1/Basics Of Programming/Patterns/Pattern 11/Pattern11_Solution.java
import java.util.*;

public class Main {

    public static void main(String[] args) {
        Scanner scn = new Scanner(System.in);

        // write ur code here
        int num=scn.nextInt();
        int n=1;
        for(int i=1;i<=num;++i){
            for(int j=1;j<=i;++j){
                System.out.print((n)+"	");
                n++;
            }
            System.out.println();
        }

    }
}